src/pages/dataTableUtilities/actions/cellActions.js

```diff
@@ -36,6 +36,28 @@ export const refreshIndicatorRow = route => ({
   payload: { route },
 });
 
+export const editSensorLocation = (location, route) => (dispatch, getState) => {
```

Comment:
this file is getting pretty big! may need to split the cellActions into different files at some point.

Reply:
definitely .. ! I don't really like 'cell' actions at all to be honest.. I would prefer not having the 'pages' reducer which is really confusing and think refactoring that into more specific `'requisitionReducer'`, `stocktakeReducer` would probably be a lot nicer!